crates/tabby/src/services/completion.rs

```diff
@@ -216,7 +227,16 @@ impl CompletionService {
     ) -> Result<CompletionResponse, CompletionError> {
         let completion_id = format!("cmpl-{}", uuid::Uuid::new_v4());
         let language = request.language_or_unknown();
-        let options = Self::text_generation_options(language.as_str());
+        let options = Self::text_generation_options(
+            language.as_str(),
+            request.temperature.unwrap_or(0.1),
+            request.seed.unwrap_or_else(|| {
```

Comment:
could you also add this seed setting logic to `/chat` api? (probably add temperature to that path as well)

Reply:
Could that be done in a separate PR?

Reply:
sounds good

Reply:
#1238